scripts/npm/link/index.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var path = require( 'path' );
+var fs = require( 'fs' );
+var componentsDir = path.join( __dirname, '..', '..', '..', 'components' );
+var packageContent = fs.readFileSync( path.join( __dirname, '..', '..', '..', 'package.json' ) );
+var packageJSON = JSON.parse( packageContent );
+var exec = require( 'child-process-promise' ).exec;
+
+function filterComponents( deps ) {
+  for ( var component in deps ) {
+    if ( /^cf-/.test( component ) ) {
```

Comment:
It could though I can add a condition to filter for that too when the time comes. Aside, that name is kind of confusing since it's not a CF component. Maybe we should rethink that first.